src/content/docs/cypher/syntax.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Syntax
+description: Basics of the Cypher query language syntax
+---
+
+In this page, we list the syntactic features of Cypher as implemented in Kùzu. As described in the
+[overview](/cypher) page, Cypher is a declarative graph query language, and Kùzu's implementation is
+based on [openCypher](https://opencypher.org/resources/).
+
+## Parsing
+
+The query parser looks for an input `STRING` that consists of ASCII or unicode characters. The parser is case-insensitive
+and ignores leading and trailing whitespaces. You can use characters from non-English languages.
+To use special characters or unicode characters as identifiers, you can escape them by encapsulating
+them in backticks \`, such as \``Ψ`\`.
+
+```cypher
+// Create a node table of books in German
+CREATE NODE TABLE `B\u00fccher` (title STRING, price INT64, PRIMARY KEY (title))
+CREATE (n:`B\u00fccher` {title: 'Der Thron der Sieben Königreiche'}) SET n.price = 20
+// Query using the unicode representation of the table name
+MATCH (n:Bücher) RETURN label(n)
+```
+```
+┌─────────┐
+│ Bücher  │
+│ STRING  │
+├─────────┤
+│ Bücher  │
+└─────────┘
+```
+
+Breaking a query into multiple lines is allowed (and recommended for readability reasons), and the
+parser ignores leading and trailing whitespaces. You can explicitly mark the end of a query with a semicolon `;`.
+
+```cypher
+MATCH (a:Person)
+WHERE a.age < 30
+RETURN a.*;
+```
+
+## Clauses
```

Comment:
Is this from Neo4j, I have a feeling that user probably doesn't need to understand the terminology of Clause.

If we decide to keep this section, is `subclause` a defined notion by Neo4j? I don't recall seeing this in the openCypher manual.

Reply:
I think clause is reasonably well understood from its role in natural language sentences, too. It's not just for query languages, so I think we can leave it there. I broke it down by section because it can be beneficial for users to know what category belongs to.

And subclause is in openCypher9, see the "Reserved keywords" section. It's also adopted by Neo4j, but I looked at the openCypher9 guide.
https://s3.amazonaws.com/artifacts.opencypher.org/openCypher9.pdf